### ISSUE TITLE
Remove unused packages?

### ISSUE THREAD

**Issue by Worthaboutapig** [jeffijoe/awilix-router-core#12]
Whilst importing awilix-koa, the latest typescript/ node defs complain about `no exported member IOptions`, which is referenced here: ["node_modules/minimatch/dist/cjs/index"' has no exported member 'IOptions'](https://stackoverflow.com/questions/75890950/node-modules-minimatch-dist-cjs-index-has-no-exported-member-ioptions).

Although that's for awilix-koa, there are two definitely-typed packages (`@types/glob` and `@types/rimraf`) that are in the `devDependencies` but don't seem to be necessary? The build compiles and tests run without them.

Do they need to be there, or can they be removed?

**Comment by jeffijoe** [comment]
Those can be removed - they were likely added way back via `typesync`.

**Comment by jeffijoe** [comment]
Yes, can be removed - yes we are using semantic release still, so that has to stay

**Comment by Worthaboutapig** [comment]
@jeffijoe I've added a PR for this, would appreciate if you could apply this too, so I can update awilix-koa to support the async controller searching.

**Comment by davidzwa** [comment]
This issue can now be closed.
